lc0_main.go

```diff
@@ -46,7 +46,7 @@ var (
 	hasDx           bool
 	testedCudnnFp16 bool
 
-	localHost = "Unknown"
```

Comment:
Keep this line a `defaultLocalHost` to use later.

Reply:
should be done